### PR TITLE
Top stories with videos

### PR DESCRIPTION
**Pull request by leggsimon**
@aintgoin2goa @ironsidevsquincy

### CODE REVIEW

File: server/lib/schema.js [hunk 1]
```diff
@@ -43,8 +43,13 @@ const queryType = new GraphQLObjectType({
 					type: new GraphQLNonNull(Region)
 				}
 			},
-			resolve: (root, { region }, { rootValue: { flags, backend = backendReal }}) =>
-				backend(flags).capi.list(sources[`${region}TopList`].uuid)
+			resolve: (root, { region }, { rootValue: { flags, backend = backendReal }}) => {
+				const listUuid = flags && flags.useVideoTopStoriesData && region === 'uk' ?
```
Comment (leggsimon) [inline server/lib/schema.js]: not 100% on if I need `&& region === 'uk'` as well here but I've only been given 1 uuid from Mus so am assuming it's going to be a uk one. Will double check with him and change if necessary.

Comment (aintgoin2goa) [inline server/lib/schema.js]: You don't need it - it's only used to select which list to use and you already know that

**Comment by aintgoin2goa** [comment]
LGTM, but is any reason why this needs to be specifically about videos? Could it just be a generic test list that we could use for other stuff, or is the plan to create a new test list for each new thing?

File: server/lib/schema.js (hunk 1 above)
Comment (leggsimon) [inline server/lib/schema.js]: Yes but I assume this will only replace the UK list for the time being?

**Comment by leggsimon** [comment]
`¯\_(ツ)_/¯` That's up to us I guess. It could be generic but I feel that this is just clearer as to what this list is being used for

File: server/lib/schema.js (hunk 1 above)
Comment (aintgoin2goa) [inline server/lib/schema.js]: That's entirely up to you - frontpage is just passing it the region based on the selected edition - you could replace both if you wanted to (which is probably better and easier)

Comment (leggsimon) [inline server/lib/schema.js]: I'll double check with editorial and remove it if necessary

**Comment by aintgoin2goa** [comment]
Fair point, I'm over-thinking this 👍

**Comment by leggsimon** [comment]
Considering I don't know of this being used before and/or a lot so I kind of think we could just implement it for each one and if it becomes bloated we'll clean it up

**Comment by leggsimon** [comment]
This way, it's quite clear what this one is

**Pull request merged**